<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DamGui_Pop_Function</class>
 <widget class="QWidget" name="DamGui_Pop_Function">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>820</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>758</width>
        <height>802</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <item>
        <widget class="QGroupBox" name="general_groupBox">
         <property name="minimumSize">
          <size>
           <width>675</width>
           <height>0</height>
          </size>
         </property>
         <property name="title">
          <string>General Parameters</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="Sys_Member_String_BaseWid" name="name" native="true">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>50</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Sys_Member_Int_BaseWid" name="id" native="true">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>50</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_1">
         <property name="minimumSize">
          <size>
           <width>675</width>
           <height>0</height>
          </size>
         </property>
         <property name="title">
          <string>Damage Function</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QFrame" name="frame_1">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>290</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="Sys_Plot_Frame" name="widget_plot" native="true"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Sys_Plot_Frame</class>
   <extends>QWidget</extends>
   <header location="global">Sys_Plot_Frame.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Sys_Member_String_BaseWid</class>
   <extends>QWidget</extends>
   <header>Sys_Member_String_BaseWid.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Sys_Member_Int_BaseWid</class>
   <extends>QWidget</extends>
   <header>Sys_Member_Int_BaseWid.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
